modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/stat/IgniteStatisticsManagerImpl.java

```diff
@@ -103,7 +105,7 @@ public IgniteStatisticsRepository statisticsRepository() {
      * @param colNames Column names.
      * @return Column with specified names.
      */
-    private Column[] filterColumns(Column[] columns, String... colNames) {
+    private Column[] filterColumns(Column[] columns, @Nullable String... colNames) {
```

Comment:
I think passing "null" to vararg is a misusage.
Method calls below are equal, in the first 2 cases a implicit empty array will be created.
method((Object)null)
method()
method(new Object[0])

Reply:
Yes, misused of Nullable annotation.